src/components/LoginPage.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { Button, Input, Row } from 'react-materialize';
+
+const LoginPage = () => ({
+  render() {
+    return (
+      <div>
+        <h1>Login page</h1>
+        <Row>
+          <Row>
+            <Input
+              type="email"
+              label="email"
+              s={12}
+            />
+          </Row>
+          <Row>
+            <Input
+              type="password"
+              label="password"
+              s={12}
+            />
+          </Row>
+          <Link to="/"><Button waves="light">Login</Button></Link>
```

Comment:
The button should not be wrapped in a link

Reply:
Alright. Will fix this